scripts/zones/Port_Jeuno/IDs.lua

```diff
@@ -33,6 +33,7 @@ zones[xi.zone.PORT_JEUNO] =
         COUNTER_NPC                   = 7077, -- I think the airships are a subtle form of pressure on the other three nations. That way, Jeuno can maintain the current balance of power.
         DEPARTURE_NPC                 = 7086, -- Have a safe flight!
         ARRIVAL_NPC                   = 7087, -- Enjoy your stay in Jeuno!
+        CUSTOMS_CAUGHT                = 7079, -- Hey, Stop right there!
```

Comment:
Is this used anywhere? The client side event already contains this 
https://discord.com/channels/907328191943823420/978719889139327026/1183108212568883351

Reply:
I forgot to take it out. ☠️